cloudbuild.yaml

```diff
@@ -9,9 +9,10 @@ steps:
       - '-f'
       - 'builder.dockerfile'
   - name: 'us.gcr.io/$PROJECT_ID/craft:builder-$COMMIT_SHA'
-  - name: 'gcr.io/kaniko-project/executor:v1.5.1'
+  - name: 'gcr.io/kaniko-project/executor:v1.12.1'
     args:
       - '--cache=true'
+      - '--compressed-caching=false'
```

Comment:
had to add this otherwise build goes OOM with error code 137
see https://github.com/GoogleContainerTools/kaniko/issues/1669#issuecomment-1260772687

Reply:
I think the problem is layer 2 is too big -- it'd probably be fine if that were split into several layers